README-template.md

```diff
@@ -118,7 +115,6 @@ repository. It can be used using the `--packages` option or the
 | Spark 3.3  | `com.google.cloud.spark:spark-3.3-bigquery:${next-release-tag}`                    |
 | Spark 3.2  | `com.google.cloud.spark:spark-3.2-bigquery:${next-release-tag}`                    |
 | Spark 3.1  | `com.google.cloud.spark:spark-3.1-bigquery:${next-release-tag}`                    |
-| Spark 2.4  | `com.google.cloud.spark:spark-2.4-bigquery:${next-release-tag}`                    |
```

Comment:
The line should remain, with a fixed `0.37.0` version (like Scala 2.11)

Reply:
Done